lib/vagrant-service-manager/command.rb

```diff
@@ -73,6 +77,79 @@ def copy_from_box(hIP, hport, husername, hprivate_key_path, source, destination)
         end
       end
 
+      def check_if_a_service_is_running(machine, service)
+        with_target_vms(nil, {:single_target=>true}) do |machine|
+          machine.communicate.execute(command) do |type, data|
+          end
+        end
+      end
+
+
+      def print_all_provider_info
+        message = <<-msg
+# Showing the status of providers in the vagrant box:
+        msg
+        @env.ui.info(message)
+        self.execute_docker_info
+        self.execute_openshift_info
+      end
+
+      def execute_openshift_info
+        @@OPENSHIFT_PORT = 8443
+        command = "systemctl status openshift"
+        with_target_vms(nil, {:single_target=>true}) do |machine|
+          if machine.communicate.test(command) then
+            # Find the guest IP
+            guest_ip = self.find_machine_ip(machine)
+            openshift_url = "https://#{guest_ip}:#@@OPENSHIFT_PORT"
+            openshift_console_url = "#{openshift_url}/console"
+            self.print_openshift_info(openshift_url, openshift_console_url)
+          else
+            @env.ui.info("# OpenShift service is not running in the vagrant box.")
+            exit 1
```

Comment:
@bexelbie 
>Error messages should probably be # preceded so that users running an $(eval) don't get a bad failure on the eval that would hide the real failure case.

But, the clients consuming the command should first check for the exit status of command before `eval`uation. #45 addresses same.